### PR TITLE
5835 arviointipyynnon perumisesta jalki arviointihistoriaan

### PR DESCRIPTION
**Pull request by TeroOtavia**
New evaluation event type added to evaluation -> cancelled evaluation request

Resolves: #5835 

Depends on https://github.com/otavanopisto/pyramus/pull/1398

### CODE REVIEW

**Comment by saaralepisto** [comment]
https://github.com/otavanopisto/pyramus/pull/1398

**Review by aviljakaine — DISMISSED** [review]

File: muikku-core-plugins-persistence/src/main/java/fi/otavanopisto/muikku/plugins/evaluation/model/AssessmentRequestCancellation.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  
+  @NotNull
+  @Column(nullable = false)
+  private Long studentEntityId;
+
+  private Long workspaceEntityId;
```
Comment (pkukkon) [inline muikku-core-plugins-persistence/src/main/java/fi/otavanopisto/muikku/plugins/evaluation/model/AssessmentRequestCancellation.java:57]: Shouldn't this be non-nullable as well?

**Review by pkukkon — CHANGES_REQUESTED** [review]

**Review by pkukkon — APPROVED** [review]
Backend OK.

**Review by jjlankinen — APPROVED** [review]
Feature tested and works fine.

**Pull request merged**